src/app/components/JumpTo/README.md

```diff
@@ -0,0 +1,12 @@
+## Description - WORK IN PROGRESS
+
+The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific sections within an article. This component renders an `H2` heading, a list of section titles as clickable `anchor` links, and groups them within a `navigation` region. When a link is clicked, the page scrolls down to the relevant section, identified by matching anchor `ids`.
```

Comment:
Can we reword this, as it's no longer a H2 heading?

"This component renders an `H2` heading, a list of section titles as clickable `anchor` links, and groups them within a `navigation` region."

Reply:
Will be done in subtask ticket